packages/core/android/src/main/kotlin/com/datadog/reactnative/DdSdkConfiguration.kt

```diff
@@ -35,6 +35,8 @@ import java.net.Proxy
  * @param proxyConfig Configuration for proxying SDK data.
  * @param serviceName Custom service name.
  * @param firstPartyHosts List of backend hosts to enable tracing with.
+ * @param bundleLogsWithRum Enables RUM for logs.
+ * @param bundleLogsWithTraces Enables Traces for logs.
```

Comment:
maybe we need to be more clear here:

```suggestion
 * @param bundleLogsWithRum Enables integration with RUM for logs.
 * @param bundleLogsWithTraces Enables integration with Traces for logs.
```